opentracing/tracer.py

```diff
@@ -21,11 +21,10 @@
 from __future__ import absolute_import
 from concurrent.futures import Future
 from .span import Span
-from .context import TraceContextSource
-from .context import TraceContextEncoder, TraceContextDecoder
+from .propagator import SpanPropagator
 
 
-class Tracer(TraceContextSource, object):
+class Tracer(SpanPropagator):
```

Comment:
I have mixed feelings about using inheritance in Python at all, but this separation makes me feel a little cleaner...